### PR TITLE
Create account

### PR DESCRIPTION
**Pull request by chantlong**
Successful Saving of Data through AJAX

### CODE REVIEW

File: src/client/components/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  handleConfirmPasswordInput() {
+    this.setState({confirmPassword: event.target.value})
+  }
+
+  createAccount(event) {
+    event.preventDefault();
+    
+    let context = this;
+    // $.post('/api/signup', function(users) {
```
Comment (mvargeson) [inline src/client/components/SignUp.js]: please remove commented code

File: src/server/routes/api.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
  */
+router.route('/api/signup')
+  .post((req, res) => {
+    console.log('da body', req.body);
+    User.create({
+      name: req.body.name,
+      email: req.body.email,
+      password: req.body.password
+    }).then(function(user) {
+      console.log('the data', user)
```
Comment (mvargeson) [inline src/server/routes/api.js]: where is your response sent?

File: src/server/routes/api.js [hunk 3]
```diff
@@ -1,17 +1,27 @@
...
 
 const router = new Router();
 
-
 /*
  * Decks
  */
+router.route('/api/signup')
+  .post((req, res) => {
+    console.log('da body', req.body);
```
Comment (mvargeson) [inline src/server/routes/api.js]: please remove console logs

File: src/client/components/SignUp.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    </div>
+    )
+  }
+}
+
+export default SignUp;
+
+
+
+// <div className="row">
```
Comment (mvargeson) [inline src/client/components/SignUp.js]: please remove commented code or implement finish this portion of the impl

File: src/client/components/SignUp.js [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+const http = require('http');
+
+class SignUp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      email: '',
+      password: '',
+      confirmPassword: ''
```
Comment (mvargeson) [inline src/client/components/SignUp.js]: currently unused

File: src/client/components/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+  handleEmailInput(event) {
+    this.setState({email: event.target.value})
+  }
+
+  handlePasswordInput(event) {
+    this.setState({password: event.target.value})
+  }
+
+  handleConfirmPasswordInput() {
```
Comment (mvargeson) [inline src/client/components/SignUp.js]: currently unused

File: src/server/routes/api.js [hunk 7]
```diff
@@ -1,17 +1,27 @@
...
+    User.create({
```
Comment (mvargeson) [inline src/server/routes/api.js]: should probably be in the controller

File: src/server/routes/api.js [hunk 8]
```diff
@@ -1,17 +1,27 @@
...
 import User from '../models/user';
 import users from '../controllers/users';
 
 const router = new Router();
 
-
 /*
  * Decks
  */
+router.route('/api/signup')
```
Comment (mvargeson) [inline src/server/routes/api.js]: should be grouped with like routes instead of under the decks

File: src/client/components/SignUp.js (hunk 5 above)
Comment (chantlong) [inline src/client/components/SignUp.js]: what's current unused? Those states are being saved.

**Pull request merged**